x-pack/solutions/security/plugins/security_solution/public/detections/index.ts

```diff
@@ -17,6 +17,7 @@ import type { StartPlugins } from '../types';
 export const DETECTIONS_TABLE_IDS: TableIdLiteral[] = [
```

Comment:
I just realized that this is a duplicate of `detectionsTimelineIds` in `x-pack/solutions/security/plugins/security_solution/public/common/components/events_viewer/use_timelines_events.tsx`